### ISSUE TITLE
Documentation for running locally

### ISSUE THREAD

**Issue by LesterThomas** [nhtua/greencam#12]
I the file https://github.com/nhtua/greencam/blob/main/docs/how-to-use-browser-in-obs.md

The instructions still point to the remote https://greencam.netlify.app/ site, but I believe you should be pointing to the local player.html file?

**Comment by syntax-tm** [comment]
Yes, I don't know why the documentation is wrong but here's the steps in case anyone else is confused by this.

1. Clone the repo
2. Switch to whatever branch you want to be using
3. Run `npm install`
4. Launch OBS
5. Add a browser source
6. Enter a name ("_GreenCam_" is recommended)
7. Check the _Local file_ checkbox
8. Browse to and select the local file
    - `better-masking-technique` and `v1` branches use `Index.html`
    - `main` branch uses the `player.html`

You might need to open the app's _.html_ file in your default browser and allow access to your webcam. Also, you can also enable hardware acceleration from [the OBS settings](https://user-images.githubusercontent.com/13120514/123204109-6da1ef00-d485-11eb-99f5-b7b4b98420a3.png) (under `Sources > Advanced`).

![image](https://user-images.githubusercontent.com/13120514/123203981-2b78ad80-d485-11eb-9fc1-84b8ad022a65.png)

**Comment by KingBain** [comment]
Unable to get this to load, I think OBS is waiting for me to allow/select my webcamera in my browser, whichever I've already done 
![image](https://user-images.githubusercontent.com/367922/135307604-7008650a-000d-4dee-94f7-07db4fd1365c.png)


the local file runs properly in chrome and edge

**Comment by nhtua** [comment]
@KingBain Looks like you need to add `--enable-gpu --enable-media-stream` into the application shortcut (Windows OS)
https://www.lifewire.com/command-line-parameters-video-games-3399930
